### PR TITLE
please accept my changes-lechu

### PR DESCRIPTION
**Pull request by lechu24**

### CODE REVIEW

File: helloWorld.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
-print "HELLO WORLD"
+print "HELLO WORLD "
+#lechu's commit
```
Comment (sriramarul) [inline helloWorld.py:2]: why did you add this here?

**Review by sriramarul — CHANGES_REQUESTED** [review]

File: helloWorld.py (hunk 1 above)
Comment (lechu24) [reply helloWorld.py:2]: just

**Review by sriramarul — APPROVED** [review]

File: helloWorld.py (hunk 1 above)
Comment (sachithub6-0) [reply helloWorld.py:2]: hello hi
